content/news/this-week-in-redox-35.md

```diff
@@ -0,0 +1,213 @@
++++
+title = "This Week in Redox 35"
+author = "goyox86"
+date = "2018-01-26 15:54:24 +0000"
++++
+
+This is the 35th post of a series of blog posts tracking the development and progress of Redox, the Rust operating system. If you want to know more about Redox in general, visit our [Github page](https://github.com/redox-os/redox).
+
+*(edited by [@goyox86](https://github.com/goyox86))*
+
+# PSA
+
+If you have any questions, ideas, or are curious about Redox, we recommend joining `#redox` on `irc.mozilla.org`, [our Discourse forum](https://discourse.redox-os.org/) or you can get an invite to our chat by sending an email request to `info@redox-os.org`.
+
+# What's new in Redox?
+
+## TL;DR
+
+Hello people! Welcome to the 35th edition of TWiRx!
+
+We started this year with [@jackpot51](https://github.com/jackpot51) being featured on the [Changelog](https://changelog.com/podcast/280) talking about building a secure operating system in Rust and about Redox in general. Make sure you listen to the episode, it's pretty good! (specially the part on which Jeremy says **SPOILER ALERT!**: "Redox is not going away")
+
+This month we've also got the [Redox Crash Challenge](https://www.reddit.com/r/rust/comments/7rv9vw/redox_os_crash_challenge/) going on. An initiative to attempt to cause the system to:
+
+- Escalate privilege.
+- Lock up.
+- Kernel panic.
+- Program crash.
+- Any other unexpected behavior.
+
+Basically, a fun way of testing and discover bugs in the system, and ultimately hardening it!
+
+Regarding changes, we have some interesting ones. Starting with the work done in the **kernel** related to the implementation of the `SIGCONT/SIGSTOP` signals, fixes for the TLS when forking that were causing some programs to misbehave and/or crash, and the implementation of `waitpid` on `PGID`. 
+
+One interesting thing I saw in the kernel was the start of the work on PTI (Page Table Isolation) as a mitigation for the recently discovered [Meltdown vulnerability](https://en.wikipedia.org/wiki/Meltdown_(security_vulnerability)). I was gladly impressed when looking at the [changes](https://github.com/redox-os/kernel/commit/a6550341bbe3614d595dd8ba8113517f3a25f637), very interesting to see how small, localized and the time frame in which they were made. One of the advantages of having a microkernel!
+
+Also in the low level, **syscall**, our system call interface crate, got a bunch of additions like: `WUNTRACED`/`WCONTINUED`, `wif*` functions, and lastly a few more `wait` calls.
+
+A quick look at **Redoxfs** reveals a small change to enter into the null namespace by [@jackpot51](https://github.com/jackpot51).
+
+Moving up to the **drivers** [@ids1024](https://github.com/ids1024) have been busy working on an `ATAPI` driver which gives support for CD-ROM drives. I think is pretty cool!
```

Comment:
I'm also working in a separate repository on iso9660, the standard filesystem for cdrom drives: https://github.com/ids1024/iso9660-rs